gem/ci/build.ps1

```diff
@@ -36,8 +36,11 @@ function Get-EnterpriseContainerVersion(
       Uri = "https://raw.githubusercontent.com/puppetlabs/enterprise-dist/$PeVer/packages.json"
       Headers = @{ "Authorization" = "token $Token" }
     }
+    $v = "" | Select-Object -Property version,release
     $packages = Invoke-RestMethod @params
-    $packages."ubuntu-18.04-amd64"."$Package"."version"
+    $v.version = $packages."ubuntu-18.04-amd64"."$Package"."version"
```

Comment:
I think you can just return a hash here like this and skip the `$v` bits:

```powershell
@{
  Version = $packages."ubuntu-18.04-amd64"."$Package"."version"
  Release = $packages."ubuntu-18.04-amd64"."$Package"."release"
}
```

Though I'm not seeing in the code where these new properties are being consumed, so I'm a bit confused.

Reply:
Ohhh... now I see over in https://github.com/puppetlabs/pe-puppetdb-extensions/pull/477/files#diff-fec826feae04e51c0d94076385408bdcR56

Reply:
Ah, cool, I'll try that out

Reply:
```
$foo = @{
>> Version = '1.2.3'
>> Release = 'ubuntu'
>> }

❯ $foo.Version
1.2.3

❯ $foo.Release
ubuntu
```